### ISSUE TITLE
Errors when importing online project

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/translationCore#4763]
0.10.0 (8024f2a)
https://git.door43.org/tc01/en_tit
This project imports correctly in 0.10.0 (db8cd53) 

![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/4c948b1a-705e-452e-925e-04703025e5cc)

**Comment by cckozie** [comment]
FWIW: I ran into the same problem when importing a local USFM2 file on 0.10.0 (cfa7277). 
![image](https://user-images.githubusercontent.com/8171759/43210791-a3dbcdc0-8ff5-11e8-81d9-fe88c368d0ab.png)

Once I got this error, I was no longer able to import usfm projects.
Tried deleting both translationCore folders and it continued to get the same error.
Then tried reinstalling tC. Still getting the error.
Deleted both translationCore folders and rebooted. Still getting the error.
Went back to 0.10.0 (db8cd53) and imports now work again.
